lib/ansible/module_utils/common/locale.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c), Ansible Project
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from ansible.module_utils._text import to_native
+
+
+def get_best_parsable_locale(module, preferences=None):
+    '''
+        Attempts to return the best possible locale for parsing output in English
+        useful for scraping output with i18n tools. When this raises an exception
+        and the caller wants to continue, it should use the 'C' locale.
+
+        :param module: an AnsibleModule instance
+        :param preferences: A list of preferred locales, in order of preference
+        :returns: The first matched preferred locale or 'C' which is the default
+    '''
+
+    locale = module.get_bin_path("locale")
+    if not locale:
+        # not using required=true as that forces fail_json
+        raise RuntimeWarning("Could not find 'locale' tool")
+
+    available = []
+    found = 'C'  # default posix, its ascii but always there
+
+    if preferences is None:
+        # new POSIX standard or English cause those are messages core team expects
+        # yes, the last 2 are the same but some systems are weird
+        preferences = ['C.utf8', 'en_US.utf8', 'C', 'POSIX']
+
+    rc, out, err = module.run_command([locale, '-a'])
```

Comment:
Do you think we should use `get_bin_path` over here?

Reply:
that is how `locale` variable gets set, see line 21

Reply:
sorry my bad. I didn't see that.